checklist/src/checklist_bot.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright 2020 The AMP HTML Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Octokit} from '@octokit/rest';
+
+import {GitHub} from './github';
+import {ILogger} from './types';
+import {checklistSectionBody, commentBody} from './template';
+
+const path = 'extensions';
```

Comment:
How about making this a regex to begin with, and not have to construct one from string concatenation? This way you can just match the part of the string you want and pull it from the matcher object
```typescript
const PATH_REGEX = /^extensions\/(.+)\/.+/;`

Reply:
Also if we want to let others use this for their own repos you can make this an env variable, but let's not jump ahead of ourselves

Reply:
I don't think we even need the environment variable, since the repo and owner names come from the PR webhooks.

Reply:
I agree, I like this suggestion